tests/e2e/ansible/group_vars/all

```diff
@@ -0,0 +1,7 @@
+---
+# Please keep alphabetically sorted.
+#
+container_runtime: containerd
+go_version: 1.16.15
+operator_sdk_version: v1.22.0
+k8s_version: v1.24.0
```

Comment:
The versions should be aligned to the ones used for building the code
go - 1.18
operator-sdk - v1.23

I think we can have have a VERSIONS file or similar to document the versions in one place.